README.md

```diff
@@ -217,9 +217,23 @@ Benchee.run(%{
 
 ```
 
-### Setup and teardown
+### Hooks (Setup, Teardown etc.)
 
-If you want to do setup and teardown, i.e. do something before or after the benchmarking suite executes this is very easy in benchee. As benchee is just plain old functions just do it before/after you call benchee:
+If you want to do something before or after something happens in benchee, we got you coverd! In general, there are 3 distrinct times where you might want to do something before/after it happened:
```

Comment:
s/distrinct/distinct/

Reply:
This gets muddy. Can we simplify before/after in terms of "hooks"?

"Benchee has three distinct [phases|stages] you can add hooks around"

Reply:
that sounds good